modules/installation/pages/2-pre-installation/3-initial-configuration.adoc

```diff
@@ -2,6 +2,13 @@
 :toc:
 :toclevels: 3
 
+This section will configure a brand new hypervisor image; first login, and network setup.
```

Comment:
This section will configure a new hypervisor installation; first login, and network setup.

Reply:
All occurrences of "the USB" should be replaced with something like "the USB drive".